unmarshal_test.go

```diff
@@ -2341,6 +2341,40 @@ func TestJSONDataHandler(t *testing.T) {
 	})
 }
 
+type Person struct {
+	FirstName string `json:"firstName"`
+	LastName  string `json:"lastName"`
+}
+
+func TestExcludeKnownFieldsFromMap(t *testing.T) {
+	t.Run("test_exclude_known_fields_from_map_with_empty_map", func(t *testing.T) {
+		p := Person{}
+		result, err := Unmarshal([]byte(`{"firstName": "string_firstName", "lastName": "string_lastName"}`), &p, WithExcludeKnownFieldsFromMap(true))
+		if err != nil {
+			t.Errorf("unexpected error %v", err)
+		}
+		if len(result) != 0 {
+			t.Errorf("failure in excluding untyped fields")
+		}
+	})
+
+	t.Run("test_exclude_known_fields_from_map", func(t *testing.T) {
+		p := Person{}
+		result, err := Unmarshal([]byte(`{"firstName": "string_firstName", "lastName": "string_lastName", "field_to_exclude": "untyped"}`), &p, WithExcludeKnownFieldsFromMap(true))
```

Comment:
same thing, i suggest rename field_to_exclude to "unknown":"string_unknown"

Reply:
fixed the naming